lib/stripe/util.rb

```diff
@@ -29,6 +29,7 @@ def self.object_classes
         'balance_transaction' => BalanceTransaction,
         'bank_account' => BankAccount,
         'card' => Card,
+        'country_spec' => CountrySpec,
```

Comment:
Sorry for the pickiness here, but would you mind keeping this list alphabetized (i.e. move "country spec" one down).